### PR TITLE
Update CPack to use tag in filename and update NSIS package

### LINKED ISSUE #1611: Installers don't remove deleted files
**Describe the bug**
If we remove files from the repo between releases, the binary installers do not remove them at install time.

**To Reproduce**
1. Install beta.6 (any os)
2. Install beta.7 over beta.6 (any os)
3. notice both libertine and libertinus fonts are still there. (we got rid of libertine for libtertinus between releases).

**Expected behavior**
Installers clean up dead files. The reproduce steps is just one example. We have changed icons/images in the themes and removed files that won't get removed. In the release after beta.7 we are going to get rid of dead themes that won't get removed as intended.

**Screenshots**
n/a

**Platform and version (please complete the following information):**
 - OS: Linux .deb, macOS .dmg and Windows .exe
 - Freeciv21 version: 3.0-beta.7
 - Ruleset/Longturn game (if applicable): n/a

**Additional context**

### PR DESCRIPTION
**Pull request by jwrober**
Closes #1611

### CODE REVIEW

File: cmake/NSIS.template.in [hunk 1]
```diff
@@ -29,8 +29,6 @@
   !define MUI_ICON "@CPACK_NSIS_MUI_ICON@"
   !define MUI_UNICON "@CPACK_NSIS_MUI_UNIICON@"
   !define MUI_UI_COMPONENTSPAGE_SMALLDESC
-  !define MUI_LICENSEPAGE_CHECKBOX
-  !define MUI_LICENSEPAGE_CHECKBOX_TEXT "I agree to the license terms."
```
Comment (lmoureaux) [inline cmake/NSIS.template.in:33]: Does this change the license page? If yes, how (might be good, just want to understand)? Do we need to update the docs screenshots?

Comment (jwrober) [reply cmake/NSIS.template.in:33]: it's been commented out for some time. I updated the docs at that time. I am now formally removing the lines.

Comment (lmoureaux) [reply cmake/NSIS.template.in:33]: Ah so ! is a comment? :D

**Review by lmoureaux — APPROVED** [review]

File: cmake/NSIS.template.in (hunk 1 above)
Comment (jwrober) [reply cmake/NSIS.template.in:33]: no there is a commented line earlier that disabled these. `!` is an include directive. `;` and `#` are comments for NSIS.

**Pull request merged**